src/libyang2/Makefile

```diff
@@ -26,6 +26,10 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	dpkg-source -x libyang2_$(LIBYANG2_FULLVERSION).dsc
 
 	pushd libyang2-$(LIBYANG2_VERSION)
+	#The package libyang2.1.148 is taken from debian trixie, which only has dpkg-dev version 1.21.22
+	#The bullseye package has dpkg-dev version 1.20.13
+	#The VS package has dpkg-dev version 1.19.8
+	sed -i 's/dpkg-dev (>= 1.22.5)/dpkg-dev (>= 1.19.8)/' debian/control
```

Comment:
I don't think this is the right way to do it. It changes both for bullseye and bookworm. Can we be specific here so that this change is applicable only for bullseye

Reply:
The current master branch has been built for bookworm platform. But, I found during compilation that we are compiling FRR for bookworm version of debian, bullseye version and VS platform also.